lang/syn/src/codegen/accounts/__cpi_client_accounts.rs

```diff
@@ -137,12 +137,12 @@ pub fn generate(accs: &AccountsStruct) -> proc_macro2::TokenStream {
 
             #(#re_exports)*
 
-            pub struct #name#generics {
+            pub struct #name #generics {
                 #(#account_struct_fields),*
             }
 
             #[automatically_derived]
-            impl#generics anchor_lang::ToAccountMetas for #name#generics {
+            impl #generics anchor_lang::ToAccountMetas for #name #generics {
```

Comment:
![image](https://user-images.githubusercontent.com/48654626/142023810-175abdea-f5a9-4fc8-95be-1b45bcf68ea4.png)

Reply:
not sure if this is correct